tests/sqlparser_hive.rs

```diff
@@ -244,17 +244,19 @@ fn parse_create_function() {
         Statement::CreateFunction {
             temporary,
             name,
-            class_name,
-            using,
+            bodies,
+            ..
         } => {
             assert!(temporary);
-            assert_eq!("mydb.myfunc", name.to_string());
-            assert_eq!("org.random.class.Name", class_name);
+            assert_eq!(name.to_string(), "mydb.myfunc");
```

Comment:
these changes look good to me -- what do you think @mobuchowski ?

Reply:
I agree.